t/lib/OpenQA/Test/Case.pm

```diff
@@ -80,7 +80,7 @@ sub find_most_recent_event {
     my ($schema, $event) = @_;
 
     my $results
-      = $schema->resultset('AuditEvents')->search({event => $event}, {limit => 1, order_by => {-desc => 'id'}});
+      = $schema->resultset('AuditEvents')->search({event => $event}, {order_by => {-desc => 'foo'}})->slice(0, 1);
```

Comment:
"foo"?

Reply:
`rows => 1` seems simpler

Reply:
Aha! That was to verify the real query. I guess I unstashed my test code by accident.